common/pinot/responseUtility.go

```diff
@@ -107,10 +107,10 @@ func ConvertSearchResultToVisibilityRecord(hit []interface{}, columnNames []stri
 		ShardID:          source.ShardID,
 		SearchAttributes: attributeMap,
 	}
-	if source.UpdateTime != 0 {
+	if source.UpdateTime > 0 {
```

Comment:
please add unit tests and also rename file to snake_case

Reply:
This file was factored out from pinot_client.go. And I've added unit tests for this file in pinot_client_test.go

Reply:
nice! we can add test case for -1

Reply:
Sure, I added a new test file for response_utility and moved the test method there. Also added new test case

Reply:
thanks Neil.